### ISSUE TITLE
'link' subplot attribute works with pyplot() but not plotly()

### ISSUE THREAD

**Issue by ko56** [JuliaPlots/Plots.jl#1124]
```julia
using Plots
plotly()
scatter(rand(10,6), layout=(3,2), link=:all)
```

The y axes are not identical.  But this works with pyplot().

**Comment by mkborregaard** [comment]
Yes, this is a known plotly bug. It's mentioned in the meta-issue but might make sense for it to have its own issue (since this should really be a priority).

**Comment by mkborregaard** [comment]
This is likely a duplicate of #850

**Comment by ko56** [comment]
I've seen the same problem, but decided to report only the 'link' one.

**Comment by daschw** [comment]
Closed by #1253
